### DISCUSSION TITLE
General Question Re: Parts vs Fields

### DISCUSSION THREAD

**Discussion by aboweFIS** [OrchardCMS/OrchardCore#16771] [Q&A] [answered]
If I am making a Content Type, for example something simple with a heading, some html content, and other miscellaneous fields...should I use the Title and HTML _Parts_ or _Fields_?
![image](https://github.com/user-attachments/assets/6f2abee2-c2a0-4aab-9a74-1e7abb847be6)

Does it make a difference/What is the difference? 

The only thing I've seen is that I can't rename the Parts (or at least I don't know how to yet), while the Fields you CAN rename.
![image](https://github.com/user-attachments/assets/7f1e8b8d-da53-4d96-b86a-93a404078f55)

Note that I am front-end and mainly just dealing with content and formatting of the sites, nothing with fancy back-end functionality.

**Comment by hishamco** [comment]
No need to reinvent the wheel, we already have many built-in Content parts OOB, so try to make use of them when it's possible. 

FYI content parts could be reusable across the content types

**Reply by aboweFIS** [reply]
Thanks but that doesn't answer my question at all.

**Reply by hishamco** [reply]
If you look closely at my reply it answers your question :)

In your case, if you are using the fields that will belong to your content type, but they can't be reusable

**Reply by aboweFIS** [reply]
So you're saying use Parts when you need them to be reusable, and Fields when you don't?

**Reply by hishamco** [reply, marked as answer]
Because fields aren't reusable. Parts could be attached to any content type, also you can add many as you want

**Reply by aboweFIS** [reply]
thank you
